jenkins/ring-install.sh

```diff
@@ -71,15 +88,22 @@ function add_source {
 }
 
 function add_source_centos {
+    local centos_version=${os_RELEASE:0:1}
     set +x
     sudo sh -c "cat <<-EOF >/etc/yum.repos.d/scality${RING_VERSION}.repo
 [scality-base]
-name=Centos6 - Scality Base
-baseurl=http://${SCAL_PASS}@packages.scality.com/stable_${RING_CODENAME}/centos/6/x86_64/
+name=Centos - Scality Base
+baseurl=http://${SCAL_PASS}@packages.scality.com/stable_${RING_CODENAME}/centos/${centos_version}/x86_64/
 gpgcheck=0
 EOF"
     set -x
-    sudo rpm -Uvh http://mirror.cogentco.com/pub/linux/epel/6/i386/epel-release-6-8.noarch.rpm
+    local epel_repo
+    if [[ $os_RELEASE =~ ^6 ]]; then
+        epel_repo='http://mirror.cogentco.com/pub/linux/epel/6/i386/epel-release-6-8.noarch.rpm'
```

Comment:
The url is the one of our documentation. :
https://docs.scality.com/display/R50/Requirements+and+Recommendations+for+Installation#RequirementsandRecommendationsforInstallation-DeclaringtheScalityrepository
It is True that we might want to use the x86_64 version, but I think that should be done in a different PR.